app/javascript/packs/image-preview.js

```diff
@@ -43,19 +51,19 @@ function imagePreviewFunction(imageId, imageTarget) {
           const ratio = this.height / this.width;
           img.width = displayWidth;
           img.height = displayWidth * ratio;
```

Comment:
Not related to this pull request specifically, but something I noticed is that the `displayWidth` value is larger than some mobile viewport sizes (at least in emulation, iPhone X is at 375px), causing some vertical skewing of the preview image when the image is shrunk to fit within the screen (due to styling of `img { max-width: 100% }`).

Reply:
Yeah, since I didn't write that code, and it's highly unlikely anyone will ever see the backstop with JS enabled, I'm not going to worry about it for now.